transformers/insolvency_resource.go

```diff
@@ -50,6 +50,48 @@ func InsolvencyResourceDaoToCreatedResponse(insolvencyResourceDao *models.Insolv
 	}
 }
 
+// AppointmentResourceDaoToAppointedResponse transforms an appointment resource dao into a response entity
+func PractitionerResourceDaosToPractitionerResourceDaosResponse(practitionerResourceDaos []models.PractitionerResourceDao) []models.CreatedPractitionerResource {
```

Comment:
I think this function name would be better without the 2nd 'Daos' - possibly something like PractitionerResourceDaosToPractitionerFilingsResponse if Filings is the only thing it'll be used for?